helm/superset/templates/deployment-worker.yaml

```diff
@@ -53,6 +53,7 @@ spec:
         app: {{ template "superset.name" . }}-worker
         release: {{ .Release.Name }}
     spec:
+      serviceAccountName: {{ .Values.serviceAccountName }}
```

Comment:
can you wrap this with an 'if' that checks whether the SA name is set, and then leave the value empty in values.yaml?

Reply:
What do you mean by SA name is set? I could check if the value is empty in values then I could skip this. Do you mean smth like:
```
{{ if .Values.serviceAccountName }}
serviceAccountName: {{ .Values.serviceAccountName }}
{{ end }}
```
But i would prefer to eliminate "if" statements to make code more clear

Reply:
Yep, like that. Allows the field to remain unset if unused

Reply:
@craig-rueda please approve